### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by c3c1l1a**
The aim of this pull request is to create controllers and views for the Blog app

## What was accomplished 
Added these routes and their respective controllers and placeholder views
image
image

`rails routes` command outputs 
image

## Checklist
-    [Linting tests](https://github.com/microverseinc/linters-config) pass locally with my changes
-    Used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
-    Documentation is done in a [professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def show; end
+end
```
Comment (alexander16108) [inline app/controllers/posts_controller.rb]: your post_controller should have  an index action as part of the project requirement  🥂

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+  def show; end
```
Comment (alexander16108) [inline app/controllers/users_controller.rb]: your user_controller should have  an index action as part of the project requirement  🥂

File: config/routes.rb [hunk 3]
```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users do
+    resources :posts
+  end
 end
```
Comment (alexander16108) [inline config/routes.rb]: the routes for your application should typically look like this 
`
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
`
for more details [visit this page on rails routing and configuring the rails router](https://guides.rubyonrails.org/routing.html)

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @c3c1l1a , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
soo close 👍🏾  💯 

 

## Highlights
clean code✔️ 
passing linters ✔️ 
met all the requirements ❌ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @c3c1l1a, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**